src/rocks_engine.h

```diff
@@ -191,6 +236,14 @@ namespace mongo {
         // protected by _identMapMutex
         uint32_t _maxPrefix;
 
+        // If _keepDataHistory is true, then the storage engine keeps all history after the stable
+        // timestamp, and WiredTigerKVEngine is responsible for advancing the oldest timestamp. If
+        // _keepDataHistory is false (i.e. majority reads are disabled), then we only keep history
+        // after
+        // the "no holes point", and WiredTigerOplogManager is responsible for advancing the oldest
+        // timestamp.
```

Comment:
Update comment to reflect Rocks implementation, not Wiredtiger.

Reply:
done, @igorcanadi , due to the difference of wt and rocksdb's ckpt strategy, wt is kind of sharp&clean ckpt, a key in wt's ckpt will contain only one version, and undetermined mvcc versions are stored in the so-called las file. while rocksdb is at the opposite side. This is the core reason why RocksEngine::supportsRecoverToStableTimestamp returns false.

This feature only affects the new feature "rolling back to raft-common point in-place", Without this feature, things can still work.